backend/testfiles/execution/stdlib/parser.dark

```diff
@@ -269,13 +272,22 @@ module TextToTextRoundtripping =
     ("'\\n'" |> roundtripCliScript) = "'\\n'"
     ("'\t'" |> roundtripCliScript) = "'\t'"
 
+    // list literal
+    ("[]" |> roundtripCliScript) = "[]"
+    ("[\"hello\"]" |> roundtripCliScript) = "[\"hello\"]"
+    ("[1L; 2L]" |> roundtripCliScript) = "[1L; 2L]"
+    // TODO: fix this, the trailing semicolon is getting removed
+    ("[1L; 2L; 3L;]" |> roundtripCliScript) = "[1L; 2L; 3L]"
```

Comment:
```suggestion
    ("[1L; 2L; 3L;]" |> roundtripCliScript) = "[1L; 2L; 3L]"
```
It's fine that the trailing semicolon is removed here -- it's an "extra" in this context.
(its existence is 'erased' during WT->PT)